deep_translator/base.py

```diff
@@ -27,6 +27,7 @@ def __init__(
         payload_key: Optional[str] = None,
         element_tag: Optional[str] = None,
         element_query: Optional[dict] = None,
+        already_translated: dict[str, str] = {},
```

Comment:
Why would you need this declaration here?

Reply:
For context, I ran the deep-translator for days. 

In my case, I ran into API errors due to LibreTranslator mirror server resets (to prevent bots from running 24/7).

I had already translated many batches, so instead of doing API calls, I added both the translated batches into `already_translated` and saved a shit ton of time (from 5 days of runtime for a full translation to less than 24 hours of runtime). 

We're talking like hundreds of API calls per hour.

Reply:
Ok but is it really needed to pass already_translated in the constructor? wouldn't it be sufficient to hide this from the user and just define it inside the class?